### PR TITLE
Bump nightly upon 84 release.

### PR DESCRIPTION
**Pull request by MihaiTabara**

### CODE REVIEW

File: api/src/shipit_api/common/config.py [hunk 1]
```diff
@@ -44,17 +44,18 @@
 # day).
 # We could have used the in-tree version, but there can be race conditions,
 # e.g. version bumped, but still no builds available.
-FIREFOX_NIGHTLY = "85.0a1"
+FIREFOX_NIGHTLY = "86.0a1"
+h
```
Comment (escapewindow) [inline api/src/shipit_api/common/config.py:48]: errant keystroke?

**Review by escapewindow — APPROVED** [review]
Thanks! Nit on the `h`

**Comment by jfx2006** [comment]
Please make sure to take PR #560 when this merges.

File: api/src/shipit_api/common/config.py (hunk 1 above)
Comment (MihaiTabara) [reply api/src/shipit_api/common/config.py]: Yes, good catch, removed it. thanks!

**Comment by MihaiTabara** [comment]
> Please make sure to take PR #560 when this merges.

Thanks for that PR, will do.

**Comment by MihaiTabara** [comment]
Merging to `master` for now, deploying to `production` after we ship 84 today to avoid breaking Ship-it.

**Pull request merged**